reference/tested-third-party-software.md

```diff
@@ -1,40 +1,15 @@
 # Tested Third Party Software
 
-## **Cluster Node OS**
+## **Recommended cluster node OS**

```

Comment:
"Recommended" is very different from "we have tested with", the latter being the core intent of this document.  We aren't making any specific recommendations here (or elsewhere), hence I recommend sticking with either the original "Cluster Node OS", or perhaps "Worker Node OS".